tests/addon/test_integration.py

```diff
@@ -8601,24 +8617,26 @@ def test_add_notes_to_block_exam_subdeck_with_some_already_in_subdeck(
         """Test that only notes not already in subdeck are moved."""
         with anki_session_with_addon_data.profile_loaded():
             ah_did = install_ah_deck()
+            deck_config = config.deck_config(ah_did)
 
             # Create a subdeck
             due_date = (date.today() + timedelta(days=5)).strftime("%Y-%m-%d")
-            subdeck_name, _ = create_block_exam_subdeck(ah_did, "Mixed Subdeck", due_date)
+            subdeck_name, _ = create_block_exam_subdeck(deck_config.anki_id, "Mixed Subdeck", due_date)
 
             # Create 3 notes in parent deck
-            deck_config = config.deck_config(ah_did)
             note1 = add_anki_note(anki_did=deck_config.anki_id)
             note2 = add_anki_note(anki_did=deck_config.anki_id)
             note3 = add_anki_note(anki_did=deck_config.anki_id)
 
             # Move 2 notes to subdeck first
-            first_added = add_notes_to_block_exam_subdeck(ah_did, subdeck_name, [note1.id, note2.id], due_date)
+            first_added = add_notes_to_block_exam_subdeck(
+                deck_config.anki_id, subdeck_name, [note1.id, note2.id], due_date
```

Comment:
:pick: We could change this and the other tests to use root_deck_id = deck_config.anki_id instead of the deck_config.anki_id